lib/phlex/elements.rb

```diff
@@ -35,6 +35,10 @@ def register_element(method_name, tag: nil, deprecated: false)
 			# frozen_string_literal: true
 
 			def #{method_name}(**attributes, &block)
+				if #{deprecated}
+					Kernel.warn("`#{method_name}` and `_#{method_name}` are deprecated and will be unsupported in Phlex 2.0. This HTML element is no longer recommended. Check out MDN web docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/#{tag}.")
```

Comment:
1) I couldn't get called method name here (eg. `param` or `_param`), using `__method__` or `__callee` returns `register_element` here or `register_void_element` on line 82.

2) I'm not sure if it makes sense to add link to MDN docs, but it explains why the method is being deprecated and redirects the user to more detailed context.